src/EasyAbp.Abp.EventBus.Boxes.Dtm.Http/EasyAbp/Abp/EventBus/Boxes/Dtm/Models/HttpDtmMessageInfoModel.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.Collections.Generic;
+using Dtmcli;
+using EasyAbp.Abp.EventBus.Boxes.Dtm.Http.EasyAbp.Abp.EventBus.Boxes.Options;
+using EasyAbp.Abp.EventBus.Boxes.Dtm.Options;
+using JetBrains.Annotations;
+using Volo.Abp;
+using Volo.Abp.EventBus.Distributed;
+
+namespace EasyAbp.Abp.EventBus.Boxes.Dtm.Models;
+
+public class HttpDtmMessageInfoModel : IDtmMessageInfoModel
+{
+    public bool EventsPublishingActionAdded { get; private set; }
+
+    public string Gid { get; set; }
+
+    public object DtmMessage { get; set; }
+
+    [NotNull] public DbConnectionLookupInfoModel DbConnectionLookupInfo { get; set; }
+
+    public List<OutgoingEventInfo> EventInfos { get; set; } = new();
+
+    private readonly IDtmMessageBuilder _dtmMessageBuilder;
```

Comment:
Please remove this line.

If you want to add some extra headers, override the `IDtmMessageManager.CreateDtmMessageInfoModel`.